bin/class-dump/dump.rb

```diff
@@ -361,7 +361,7 @@
 File.open(UMBRELLA_HEADER, "w:UTF-8") do |file|
   file.puts("// Generated by bin/class-dump/dump.rb")
   file.puts("")
-  file.puts("#ifndef XCT_UI_TESTING_AVAILABLE")
+  file.puts("#ifndef LOAD_XCTEST_PRIVATE_HEADERS")
```

Comment:
@jmoody Should be at least the same as in umbrella header?
In CBX-XCTest-Umbrella.h:
```
#if LOAD_XCTEST_PRIVATE_HEADERS
...
```